modules/virt-about-conditions-and-events.adoc

```diff
@@ -5,7 +5,7 @@
 [id="virt-about-conditions-and-events.adoc_{context}"]
 = About conditions and events
 
-Diagnose DataVolume issues by examining the output of the `Conditions` and `Events` sections
+Diagnose data volume issues by examining the output of the `Conditions` and `Events` sections
```

Comment:
I do find "data volume" to seem overly generic. My first thought is to wonder if we should mandate always writing it out like this:

```
`DataVolume` object(s)
```

But that would seems too wordy, especially in places where we have pluralized "DataVolume" in the past.  

And in this particular example, "Diagnose `DataVolume` object issues" sounds awkward. "Issues with `DataVolume` objects" also sounds awkward.

My second thought is... can we get away with just writing it "datavolume(s)" without the space, when we are not referring to a specific object?